packages/manager/modules/pci/src/projects/projects.routing.js

```diff
@@ -53,12 +47,20 @@ export default /* @ngInject */ ($stateProvider) => {
         return promise;
       },
       guideUrl: () => GUIDES_URL,
-      projects: /* @ngInject */ (OvhApiCloudProject) =>
-        OvhApiCloudProject.v6()
-          .query()
-          .$promise.then((projects) =>
-            map(projects, (serviceName) => new Project({ serviceName })),
-          ),
+      projects: /* @ngInject */ (PciProjectsService) =>
+        PciProjectsService.getProjects().then((projects) =>
+          projects.sort((project1, project2) => {
```

Comment:
not already handled by iceberg ?

Reply:
We need to sort the project by 2 values of the field here. We need to bring the projects in `suspended` state or `pendingDebt` state to the top, which I guess is not possible in iceberg as of today